{"timestamp":1729532384086,"message":"Unable to load MUSE MARC records","log.level":"ERROR","logger.name":"processes.muse","thread.id":140661559397248,"thread.name":"MainThread","process.id":2334,"process.name":"MainProcess","file.name":"/home/runner/work/drb-etl-pipeline/drb-etl-pipeline/processes/muse.py","line.number":122,"entity.type":"SERVICE"}
